packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -502,93 +560,97 @@ function BackgroundSizeToolsPanelItem( {
-				url={ getResolvedThemeFilePath( imageValue, themeFileURIs ) }
-				value={ backgroundPositionToCoords( positionValue ) }
-				onChange={ updateBackgroundPosition }
-			/>
-			<ToggleGroupControl
-				size="__unstable-large"
-				label={ __( 'Size' ) }
-				value={ currentValueForToggle }
-				onChange={ updateBackgroundSize }
-				isBlock
-				help={ backgroundSizeHelpText(
-					sizeValue || defaultValues?.backgroundSize
-				) }
+		<Tabs>
+			<Tabs.TabList>
+				<Tabs.Tab tabId="size">{ __( 'Style' ) }</Tabs.Tab>
+				<Tabs.Tab tabId="position">{ __( 'Position' ) }</Tabs.Tab>
+			</Tabs.TabList>
+			<Tabs.TabPanel
+				tabId="size"
+				focusable={ false }
+				className="block-editor-global-styles-background-panel__tab-panel"
 			>
-				<ToggleGroupControlOption
-					key="cover"
-					value="cover"
-					label={ _x(
-						'Cover',
-						'Size option for background image control'
-					) }
-				/>
-				<ToggleGroupControlOption
-					key="contain"
-					value="contain"
-					label={ _x(
-						'Contain',
-						'Size option for background image control'
+				<VStack spacing={ 3 } className="single-column">
+					<HStack
+						className="block-editor-global-styles-background-panel__size-controls"
+						spacing={ 4 }
+						justify="space-between"
+						alignment="top"
+					>
+						<SelectControl
+							__nextHasNoMarginBottom
+							__next40pxDefaultSize
+							label={ __( 'Size' ) }
+							options={ [
+								{
+									label: _x(
+										'Cover',
+										'Size option for background image control'
+									),
+									value: 'cover',
+								},
+								{
+									label: _x(
+										'Contain',
+										'Size option for background image control'
+									),
+									value: 'contain',
+								},
+								{
+									label: _x(
+										'Tile',
+										'Size option for background image control'
+									),
+									value: 'auto',
+								},
+							] }
+							value={ currentValueForToggle }
+							onChange={ updateBackgroundSize }
+						/>
+
+						{ currentValueForToggle === 'auto' && (
+							<UnitControl
+								label={ __( 'Width' ) }
+								aria-label={ __( 'Background image width' ) }
+								onChange={ updateBackgroundSize }
+								value={ sizeValue }
+								size="__unstable-large"
+								__unstableInputWidth="100px"
+								min={ 0 }
+								placeholder={ __( 'Auto' ) }
+							/>
+						) }
+					</HStack>
+					{ currentValueForToggle !== 'cover' && (
+						<ToggleControl
+							label={ __( 'Repeat' ) }
+							checked={ repeatCheckedValue }
+							onChange={ toggleIsRepeated }
+						/>
 					) }
-				/>
-				<ToggleGroupControlOption
-					key="tile"
-					value="auto"
-					label={ _x(
-						'Tile',
-						'Size option for background image control'
+				</VStack>
+			</Tabs.TabPanel>
+			<Tabs.TabPanel
+				tabId="position"
+				focusable={ false }
+				className="block-editor-global-styles-background-panel__tab-panel"
+			>
+				<FocalPointPicker
```

Comment:
One thing to note about the focal point picker is that its dimensions vary - it can be very tall or short depending on the image.